.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -8,26 +8,23 @@ body:
       description: Please search to see if an issue already exists for the bug you encountered.
       options:
         - label: I have searched the existing issues
-    validations:
-      required: true
+          required: true
 
   - type: checkboxes
     attributes:
       label: This report is about a single actionable bug.
       description: Please create one issue per bug, split up your bug reports if needed.
       options:
         - label: I'm reporting a single actionable bug
-    validations:
-      required: true
+          required: true
 
   - type: checkboxes
     attributes:
       label: This report is about the ProGit book, version 2, English language.
       description: Bug reports about translations or the first version of the book are not accepted.
       options:
         - label: This bug is not about a translation or old version
-    validations:
-      required: true
+          required: true
 
   - type: checkboxes
```

Comment:
Do we _require_ the following checkboxes to be ticked before proceeding?

- [ ] I confirm the bug is about the book as found on the website/pdf
- [ ] This bug also affects the Pro Git book as published on the website

Let me know if I need to require one or both of these checkboxes as well, and I'll include those changes in this PR.
That way we fix all checkboxes at once. 😄

Reply:
Nah, let them be optional for now. Ready to merge?

Reply:
Let's keep those optional then.

Reply:
@ben We're good to merge!